File: pertemuan-4/pertemuan-4/program-kelulusan-mahasiswa.js
let nama = prompt("Masukkan Nama: ");
let nilai = prompt("Masukkan Nilai: ");
let status = "";

if (nilai >= 60) {
    status = "Lulus";
} else {
    status = "Tidak Lulus";
}

console.log("Nama: " + nama);
console.log("Nilai: " + nilai);
console.log("Status Kelulusan: " + status);

File: pertemuan-4/pertemuan-4/program-kelulusan-mahasiswa.c
#include <stdio.h>

int main() {
    char nama[] = "";
    char status[] = "";
    float nilai;
    
    printf("Masukkan nama: ");
    scanf("%s", &nama);
    printf("Masukkan nilai: ");
    scanf("%f", &nilai);
    
    if (nilai >= 60) {
        strcpy(status, "Lulus");
    } else {
        strcpy(status, "Tidak Lulus");
    }
    
    printf("Nama = %s \n", nama);    
    printf("Nilai = %f \n", nilai);
    printf("Status Kelulusan = %s \n", status);

    return 0;
}